### PR TITLE
feat: TG-949 Sunbird 3.8.0 GA fixes

### PR DESCRIPTION
**Pull request by keshavprasadms**
- Roles purged as duplicates - https://github.com/project-sunbird/sunbird-learning-platform/pull/1433, https://github.com/project-sunbird/sunbird-learning-platform/pull/1434 https://github.com/project-sunbird/sunbird-data-pipeline/pull/1283
- FTL files moved from sunbird-auth repo to remove dependecy
- `play_hosts` deprectaed and replaced with `ansible_play_hosts`
- Removed all references and files of badger service
- Renamed `log-es-2` to `log-es-1` across repo
- Removed unused vars related to swarm
- Streamlined multiple repeated variables to use one single variable for API token
- Updated postgres role and connection strings to support Ubuntu 18+
- cassandra-db-update role changes to pass multiple cql files from jenkins in one shot
- Bootstrap to create flink namespaces also
- Moved cert es indexes from sunbird-utils repo to devops repo
- Added compositesearch index and mapping
- Added alias creation to ES index during provision
- Remove unused keycloak 6, postgres-master and postgres-slave roles
- Implemented handlers for keycloak bootstrap and deploy tasks
- Updated keycloak relam to be compatible with keyclaok 7
- All clients are now created as part of realm creation (including lms and other clients) 
- Removed varaible related to `fcm token` across repo
- Disabled report service health check since it's not implemented
- Updated jenkins server setup script, plugins, m2, jenkins jobs and versions of few packages
- Submodule initilization for nginx build
- Kong migrations for first boot (only when setting up the 0.14.1 version), not a upgrade from older version
- Optional variable to specify kibana version, in case we want to use log and app ES in same cluster
- Updated default alert mailing list and removed analytics vm endpoint from scrape
- Renamed few jenkins jobs and corresponding charts
- Removed reference to sunbird-auth repo during keycloak deploy
- Cleaned up and sorted private repo template inventory (variables and hosts)

### CODE REVIEW

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/inventory/env/group_vars/all.yml [hunk 1]
```diff
@@ -666,3 +618,4 @@ cassandra_cluster_size: "{{ groups['cassandra'] | length }}"
 # This value is constant for sendgrid api authentication.
 # If you're using any other mail server provider, override this value in common.yaml.
 mail_server_username: "apikey"
+bootstrap_namespace: "{{ env }},flink-{{ env }},flink-kp-{{ env }}"
```
[the remark this answers is not in the document]
Comment (keshavprasadms) [reply ansible/inventory/env/group_vars/all.yml:621]: Requires changes in playbook level also and retesting, this implementation should be fine for now. Matches the jenkins input and is in accordance with many roles we have created and used this format.

File: ansible/inventory/env/group_vars/all.yml [hunk 2]
```diff
@@ -222,9 +212,7 @@ postgresql_hba_entries:
   - { type: local, database: all, user: all, auth_method: peer }
   - { type: host, database: all, user: all, address: '127.0.0.1/32', auth_method: md5 }
   - { type: host, database: all, user: all, address: '::1/128', auth_method: md5 }
```
[the remark this answers is not in the document]
Comment (keshavprasadms) [reply ansible/inventory/env/group_vars/all.yml:214]: done

File: ansible/inventory/env/group_vars/all.yml [hunk 3]
```diff
@@ -134,8 +125,8 @@ sunbird_auth_version: 1.0v
 
 ## Content Repo configuration
 sunbird_api_auth_token: "{{ core_vault_sunbird_api_auth_token }}" #Authorization key (JWT) to access Sunbird APIs. This will be in the output of deploy-apis.sh script, extracting it out is documented in the deployment wiki.
-sunbird_ekstep_api_key: "{{ core_vault_sunbird_ekstep_api_key }}" #Authorization key (JWT) to access Ekstep APIs. Steps to generate this are documented on {{proto}}://github.com/project-sunbird/sunbird-commons/wiki/Obtaining-API-token-for-accessing-ekstep-APIs
-sunbird_plugin_repo_api_key: "{{ core_vault_sunbird_ekstep_api_key }}"
+sunbird_ekstep_api_key: "{{ core_vault_sunbird_api_auth_token }}" #Authorization key (JWT) to access Ekstep APIs. Steps to generate this are documented on {{proto}}://github.com/project-sunbird/sunbird-commons/wiki/Obtaining-API-token-for-accessing-ekstep-APIs
```
[the remark this answers is not in the document]
Comment (keshavprasadms) [reply ansible/inventory/env/group_vars/all.yml:128]: done

File: ansible/roles/postgresql-data-update/tasks/main.yml [hunk 4]
```diff
@@ -3,12 +3,6 @@
     name: python-psycopg2
     state: present
 
-- name: Verifying db users are present else creating them
```
[the remark this answers is not in the document]
Comment (keshavprasadms) [reply ansible/roles/postgresql-data-update/tasks/main.yml:6]: Handled with a softlink - Please check these
https://github.com/project-sunbird/sunbird-devops/pull/2507/commits/c1ea970fbaf6940e3c0b129db42641ebb275bca4
https://github.com/project-sunbird/sunbird-devops/pull/2507/commits/960659c3c1fd7c34e2a974ef6d258af9d8d7dc7e

File: ansible/inventory/env/group_vars/all.yml [hunk 5]
```diff
@@ -361,82 +342,59 @@ grafana_admin_password: "{{core_vault_grafana_admin_password}}"
...
   - name: "{{enc_postgres_user}}"
-    login_host: "{{enc_postgres_host}}"
     login_password: "{{enc_postgres_password}}"
     password: "{{enc_postgres_password}}"
     db: "{{enc_postgres_database}}"
     login_user: "{{enc_postgres_user}}"
     priv: "ALL"
 
-
 postgresql_databases:
```
[the remark this answers is not in the document]
Comment (keshavprasadms) [reply ansible/inventory/env/group_vars/all.yml:359]: Please check https://github.com/project-sunbird/sunbird-devops/pull/2507#discussion_r636762226

**Review by rjshrjndrn — APPROVED** [review]

**Comment by rjshrjndrn** [comment]
Nice work @keshavprasadms :tada:

**Pull request merged**